internal/cli/atlas/cloudproviders/accessroles/aws/disable.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const disableTemplate = "AWS IAM role successfully disabled.\n"
+
+type DisableOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store  store.CloudProviderAccessRoleDisabler
+	roleID string
+}
+
+func (opts *DisableOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DisableOpts) Run() error {
+	err := opts.store.DisableCloudProviderAccessRoles(opts.newCloudProviderDeauthorizationRequest())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(nil)
+}
+
+func (opts *DisableOpts) newCloudProviderDeauthorizationRequest() *atlas.CloudProviderDeauthorizationRequest {
+	return &atlas.CloudProviderDeauthorizationRequest{
+		ProviderName: provider,
+		GroupID:      opts.ConfigProjectID(),
+		RoleID:       opts.roleID,
+	}
+}
+
+// mongocli atlas cloudProvider aws accessRoles disable --roleId roleId [--projectId projectId]
```

Comment:
[q] should this be 
```suggestion
// mongocli atlas cloudProvider aws accessRoles deauthorize <roleId> [--projectId projectId]
```
?

Reply:
I decided to use `enable/disable` because we use it for other commands but it is fine with me using `deauthorize`

Reply:
It's going to depend on context and what the UI/UX uses, enable and authorize may not carry the same meaning for some users or some intentions of UX.
That said my preference here is to align to GUI